src/tactic/induction.lean

```diff
@@ -1213,7 +1213,12 @@ remember the value of the major premise.
 meta def eliminate_expr (generate_induction_hyps : bool) (major_premise : expr)
   (eq_name : option name := none) (gm := generalization_mode.generalize_all_except [])
   (with_names : list name := []) : tactic unit := do
-  num_reverted ← revert_reverse_dependencies_of_hyp major_premise,
+  ctx ← local_context,
+  revdeps ← ctx.mfilter $ λ h, do {
+    H ← infer_type h,
+    kdepends_on H major_premise
+  },
```

Comment:
This seems to match closer with the style in other files. Although the `do { }` is consistent within this file, so please choose the style you prefer.
```suggestion
  revdeps ← ctx.mfilter (λ h, do
    H ← infer_type h,
    kdepends_on H major_premise),
```

Reply:
Thanks for the suggestion (and the review in general!). I prefer the braces style, so I'll keep that for now. I think I've seen both in mathlib.